examples/visual-tests/font.amp/font.amp.html

```diff
@@ -0,0 +1,105 @@
+  </p>
+  <p>
+    Quisque ultricies id augue a convallis. Vivamus euismod est quis tellus laoreet lacinia. In quam tellus, mollis nec porta eget, volutpat sit amet nibh. Duis ac odio sem. Sed consequat, ante gravida fringilla suscipit, libero libero ullamcorper metus, nec porta est elit at est. Curabitur vel diam ligula. Nulla bibendum malesuada odio.
+  </p>
+  <p class="comic-amp-bold">
+    Pellentesque ultricies quam diam, sit amet sollicitudin ante imperdiet a. Pellentesque porta semper nisi, et lobortis est laoreet ac. Vivamus pulvinar egestas purus, vitae pharetra nisl mattis at. Duis gravida ac quam vel commodo. Pellentesque dignissim luctus magna, a fermentum ligula porttitor non. Duis sodales interdum urna, eu viverra elit dapibus vitae. Sed aliquet magna non erat suscipit, vel elementum nisl lacinia. Lorem ipsum dolor sit amet, consectetur adipiscing elit. In eu tempor purus. Cras non libero vehicula, finibus arcu eget, eleifend nulla. Vestibulum non sollicitudin turpis. Fusce condimentum posuere risus nec congue. Curabitur aliquam lorem dolor.
+  </p>
+  <p>
+    Sed mollis, justo ac volutpat gravida, nisl eros imperdiet massa, sed aliquam nulla odio in urna. Pellentesque eros urna, rutrum vel luctus id, ultrices a libero. Morbi ante justo, gravida et velit at, convallis tristique metus. Nam elementum luctus facilisis. Vestibulum nec augue dignissim, gravida odio nec, volutpat libero. Cras vitae sagittis tellus, sed tincidunt ipsum. Maecenas et mauris id nunc porttitor tempus sit amet at libero.
+  </p>
+  <p class="comic-amp-bold">
+    Aenean ullamcorper risus quam, molestie sodales lacus volutpat at. Nunc vulputate est ut faucibus faucibus. Proin posuere viverra vestibulum. Vestibulum pretium nunc ut euismod sollicitudin. Etiam ornare posuere libero. Vestibulum urna massa, viverra sed ullamcorper ac, hendrerit sed dui. Vestibulum interdum lectus tellus, ut consequat quam pulvinar vitae. Mauris porttitor nulla porta urna convallis accumsan. Curabitur eget ante in libero fringilla elementum. Curabitur tempus arcu massa, gravida tristique erat convallis vitae. Sed lacinia elit justo, eget mollis dui vehicula sit amet. Nunc dignissim condimentum nunc, id pulvinar purus mattis ut. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque lacinia nibh ac enim laoreet imperdiet.
+  </p>
+
+  <amp-font
+    layout="nodisplay"
+    font-family="Comic AMP"
+    timeout="2000"
+    on-error-remove-class="comic-amp-font-loading"
```

Comment:
I've set things up so that any JS errors seen during page load are raised by the diff tool, and yet, I didn't see any errors. Could it be that these errors occur after the page is loaded, by which time the snapshot has been taken, and the page has been closed?

Here are debug logs:

```
GET /examples/visual-tests/font.amp/font.amp.html 200 5.754 ms - 6064
2017-07-11T17:27:38 [DEBUG] WebPage - updateLoadingProgress: 50
GET /dist/v0/amp-font-0.1.max.js 200 5.180 ms - 188713
2017-07-11T17:27:38 [DEBUG] WebPage - setupFrame ""
2017-07-11T17:27:38 [DEBUG] WebPage - evaluateJavaScript "(function() { return (function () {\n      return typeof __poltergeist;\n    })(); })()"
2017-07-11T17:27:38 [DEBUG] WebPage - evaluateJavaScript result QVariant(QString, "undefined")
GET /dist/amp.js 200 6.956 ms - 1210065
2017-07-11T17:27:38 [DEBUG] WebPage - updateLoadingProgress: 100
Powered by AMP ⚡ HTML – Version 1499807954031 http://localhost:8000/examples/visual-tests/font.amp/font.amp.html
2017-07-11T17:27:38 [DEBUG] WebPage - setupFrame ""
2017-07-11T17:27:38 [DEBUG] WebPage - setupFrame ""
GET /examples/visual-tests/font.amp/fonts/ComicAMP.ttf 200 1.260 ms - 20652
2017-07-11T17:27:38 [DEBUG] WebPage - evaluateJavaScript "(function() { return (function () {\n      return typeof __poltergeist;\n    })(); })()"
2017-07-11T17:27:38 [DEBUG] WebPage - evaluateJavaScript result QVariant(QString, "object")
GET /examples/visual-tests/font.amp/fonts/ComicAMPBold.ttf 200 1.568 ms - 26436
2017-07-11T17:27:38 [DEBUG] WebPage - evaluateJavaScript "(function() { return (function() { var page_id = arguments[0]; var args = []; for(var i=1; i < arguments.length; i++){ if ((typeof(arguments[i]) == 'object') && (typeof(arguments[i]['ELEMENT']) == 'object')){ args.push(window.__poltergeist.get(arguments[i]['ELEMENT']['id']).element); } else { args.push(arguments[i]) } } var _result = (function (name, args) {\n      return __poltergeist.externalCall(name, args);\n    }).apply(this, args); return window.__poltergeist.wrapResults(_result, page_id); })(1,\"find\",[\"css\",\".i-amphtml-loader-dot\"]); })()"
2017-07-11T17:27:38 [DEBUG] WebPage - evaluateJavaScript result QVariant(QVariantMap, QMap(("value", QVariant(QVariantList, ()))))
[percy][DEBUG] Using filesystem_loader to discover assets.
[percy][DEBUG] Snapshot started (name: "Fonts")
2017-07-11T17:27:38 [DEBUG] WebPage - setupFrame ""
2017-07-11T17:27:38 [DEBUG] WebPage - evaluateJavaScript "(function() { return (function () {\n      return typeof __poltergeist;\n    })(); })()"
2017-07-11T17:27:38 [DEBUG] WebPage - evaluateJavaScript result QVariant(QString, "object")
2017-07-11T17:27:38 [DEBUG] WebPage - evaluateJavaScript "(function() { return (function() { var page_id = arguments[0]; var args = []; for(var i=1; i < arguments.length; i++){ if ((typeof(arguments[i]) == 'object') && (typeof(arguments[i]['ELEMENT']) == 'object')){ args.push(window.__poltergeist.get(arguments[i]['ELEMENT']['id']).element); } else { args.push(arguments[i]) } } var _result = (function (name, args) {\n      return __poltergeist.externalCall(name, args);\n    }).apply(this, args); return window.__poltergeist.wrapResults(_result, page_id); })(1,\"currentUrl\",[]); })()"
2017-07-11T17:27:38 [DEBUG] WebPage - evaluateJavaScript result QVariant(QVariantMap, QMap(("value", QVariant(QString, "http://localhost:8000/examples/visual-tests/font.amp/font.amp.html"))))
[percy][DEBUG] Snapshot resource: /examples/visual-tests/font.amp/font.amp.html
[percy][DEBUG] All snapshot resources loaded (0.00267s)
[percy][DEBUG] All snapshot resources uploaded (0.380223s)

[percy] Visual diffs are now processing: https://percy.io/ampproject/amphtml/builds/283488

Done! Percy snapshots are now processing...
--> https://percy.io/ampproject/amphtml/builds/283488
```